xml/System.Windows.Navigation/NavigationService.xml

```diff
@@ -76,14 +67,13 @@
+ Visually, XBAPs use Internet Explorer as a navigator, to provide an integrated user experience. Physically, however, XBAPs actually use <xref:System.Windows.Navigation.NavigationWindow> as the navigator; the <xref:System.Windows.Application.MainWindow%2A> property of an XBAP running in Internet Explorer will return a reference to the <xref:System.Windows.Navigation.NavigationWindow>, and the navigation history that is managed by the <xref:System.Windows.Navigation.NavigationWindow> is integrated with the navigation history that is managed by Internet Explorer in the following ways:
 
--   When entries in the Internet Explorer 7 navigation UI are selected, Internet Explorer 7 causes <xref:System.Windows.Navigation.NavigationService> to navigate back or forwards to the content that is associated with those entries.
+-   When content is navigated to by calling <xref:System.Windows.Navigation.NavigationService.Navigate%2A>, <xref:System.Windows.Navigation.NavigationService.GoBack%2A>, and <xref:System.Windows.Navigation.NavigationService.GoForward%2A> from within an XBAP the relevant navigation history entries are also added to Internet Explorer navigation history.
+-   When entries in the Internet Explorer navigation UI are selected, Internet Explorer causes <xref:System.Windows.Navigation.NavigationService> to navigate back or forwards to the content that is associated with those entries.
 
 > [!NOTE]
->  A <xref:System.Windows.Controls.Frame> can provide its own navigation history, or use the navigation history of the navigator that hosts it. If <xref:System.Windows.Controls.Frame> provides its own navigation history, it can display its own navigation UI to navigate through the entries in navigation history; those entries are not added to the navigation history of the host navigator (<xref:System.Windows.Navigation.NavigationWindow>, <xref:System.Windows.Controls.Frame>, Internet Explorer) and, consequently, cannot be navigated to from the navigation UI of each (see <xref:System.Windows.Controls.Frame.JournalOwnership%2A>).
+>  A <xref:System.Windows.Controls.Frame> can provide its own navigation history, or use the navigation history of the navigator that hosts it. If <xref:System.Windows.Controls.Frame> provides its own navigation history, it can display its own navigation UI to navigate through the entries in navigation history; those entries are not added to the navigation history of the host navigator (<xref:System.Windows.Navigation.NavigationWindow>, <xref:System.Windows.Controls.Frame>, browser) and, consequently, cannot be navigated to from the navigation UI of each (see <xref:System.Windows.Controls.Frame.JournalOwnership%2A>).
```

Comment:
```suggestion
>  A <xref:System.Windows.Controls.Frame> can provide its own navigation history, or use the navigation history of the navigator that hosts it. If <xref:System.Windows.Controls.Frame> provides its own navigation history, it can display its own navigation UI to navigate through the entries in navigation history; those entries are not added to the navigation history of the host navigator (<xref:System.Windows.Navigation.NavigationWindow>, <xref:System.Windows.Controls.Frame>, browser) and, consequently, cannot be navigated to from the navigation UI of each (see <xref:System.Windows.Controls.Frame.JournalOwnership%2A>).

For more information about XBAP support, see [Frequently asked questions about WPF browser-hosted applications (XBAP)](/dotnet/desktop/wpf/app-development/xbap-faq?view=netframeworkdesktop-4.8).
```